src/core/bluetoothreceiver.cpp

```diff
@@ -0,0 +1,162 @@
+/***************************************************************************
+ bluetoothreceiver.cpp - BluetoothReceiver
+
+ ---------------------
+ begin                : 18.11.2020
+ copyright            : (C) 2020 by David Signer
+ email                : david (at) opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "bluetoothreceiver.h"
+
+#include <QSettings>
+#include <QDebug>
+
+BluetoothReceiver::BluetoothReceiver( QObject *parent ) : QObject( parent ),
+  mLocalDevice( new QBluetoothLocalDevice ),
```

Comment:
```suggestion
  mLocalDevice( std::make_unique<QBluetoothLocalDevice>() ),
```

This makes it clear that it's a unique_ptr (without having to look it up during a review)